Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/notification/ShareTestResultNotificationService.kt

```diff
@@ -27,48 +26,20 @@ class ShareTestResultNotificationService @Inject constructor(
 
     fun setup() {
         Timber.d("setup()")
-
-        coronaTestRepository.coronaTests
-            .onEach { tests ->
-
-                // schedule reminder if test wasn't submitted
-                tests.filter { test ->
-                    test.isSubmissionAllowed && !test.isSubmitted
-                }.forEach { test ->
-                    maybeScheduleSharePositiveTestResultReminder(test.type)
-                }
-
-                // cancel the reminder when test is submitted
-                tests
-                    .filter { it.isSubmitted }
-                    .forEach { notification.cancelSharePositiveTestResultNotification(it.type) }
-            }
-            .catch { Timber.e(it, "Failed to schedule positive test result reminder.") }
-            .launchIn(appScope)
-
+        // Schedule positive test reminders
```

Comment:
Is this comment necessary? Function name is already very descriptive :)

Reply:
Addressed in https://github.com/corona-warn-app/cwa-app-android/pull/3074